lib/cocoapods/installer/analyzer.rb

```diff
@@ -324,21 +324,25 @@ def embedded_target_pod_targets_by_host(aggregate_target, embedded_aggregate_tar
       #
       def analyze_host_targets_in_podfile(aggregate_targets, embedded_aggregate_targets)
         target_definitions_by_uuid = {}
+        cli_host_with_frameworks = []
+        cli_product_type = 'com.apple.product-type.tool'
         # Collect aggregate target definitions by uuid to later lookup host target
         # definitions and verify their compatiblity with their embedded targets
         aggregate_targets.each do |target|
-          target.user_targets.map(&:uuid).each do |uuid|
-            target_definitions_by_uuid[uuid] = target.target_definition
+          target.user_targets.each do |user_target|
```

Comment:
can we not do `cli_host_with_frameworks = target.user_targets.map(&:uuid).select do ... end`?

Reply:
ah nm I see this does two things